src/main/java/gov/nasa/jpl/mbee/mdk/mms/MMSArtifact.java

```diff
@@ -2,11 +2,22 @@
 
 import org.apache.http.entity.ContentType;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.InputStream;
 
 public interface MMSArtifact {
     String getId();
     String getChecksum();
-    InputStream getInputStream();
+    default InputStream getInputStream() {
+        try {
```

Comment:
Why the exception suppression? Typically if there is an exception we'd like to surface it instead of having it silently fail.

Reply:
I see it was pulled up from an implementation. Lacking information as to why it was there to begin with (it's been heavily refactored so `git blame` doesn't help us without a lot of digging), we'd be better off not bubbling up.